test/controllers/APIControllerSpec.scala

```diff
@@ -62,14 +62,29 @@ class APIControllerSpec extends SpecBase {
     }
 
     s"return OK (${Status.OK}) when the call returns some data and can be parsed to the correct response" in new Setup {
+      when(mockETMPService.getNumberOfEstimatedPenalties(mockETMPPayloadForAPIResponseData)).thenReturn(2)
       when(mockETMPService.getPenaltyDataFromETMPForEnrolment(ArgumentMatchers.any())(ArgumentMatchers.any()))
         .thenReturn(Future.successful((Some(mockETMPPayloadForAPIResponseData), Right(GetETMPPayloadSuccessResponse(mockETMPPayloadForAPIResponseData)))))
       val result = controller.getSummaryDataForVRN("123456789")(fakeRequest)
       status(result) shouldBe Status.OK
       val apiDataToReturn: APIModel = APIModel(
-        mockETMPPayloadForAPIResponseData.pointsTotal
+        noOfPoints = 4,
+        noOfEstimatedPenalties = 2
       )
       contentAsString(result) shouldBe Json.toJson(apiDataToReturn).toString()
     }
+
+    s"return OK (${Status.OK}) when there are no LSP or LPP estimated penalties in etmpPayload" in new Setup {
+      when(mockETMPService.getNumberOfEstimatedPenalties(mockETMPPayloadWithNoEstimatedPenaltiesForAPIResponseData)).thenReturn(0)
```

Comment:
Sorry, could you change this to `ArgumentMatchers.any()`
```suggestion
      when(mockETMPService.getNumberOfEstimatedPenalties(ArgumentMatchers.any())).thenReturn(0)
```

Reply:
Done.